### ISSUE TITLE
Use tracks

### ISSUE THREAD

**Issue by mtxellrb** [aertslab/create_cisTarget_databases#17]
Hi,

Maybe I'm getting this totally wrong, but from the README file it seems that to annotate regulatory regions for each gene or region, you can either use a motif annotation generated by cluster-buster or you can use Chip-Seq tracks instead. However, the description seems to be focused entirely on motif annotation. Could you be so kind to provide me with an pipeline example for bigWig files of TF ChIP-seq data and gene fasta files? Thanks!

Best,

Meritxell

**Comment by ghuls** [comment]
Yes for now the README is focused on motifs. For tracks the script still needs to be written but conceptually it is quite similar to https://github.com/aertslab/create_cisTarget_databases/blob/master/create_cistarget_motif_databases.py, but instead of using scoring motifs with Cluster-buster for a FASTA file with regions/genes/ of interest, you need to have a BED file with your regions/genes and use bigWigAverageOverBed to get the max score per region and rank those. I might look at this code soon as I have to generate some databases myself.

**Comment by wariobrega** [comment]
@ghuls  I am also trying to understand whether I can use peak files generated from my ChiPSeq data!

**Comment by ghuls** [comment]
yes you can, but you need to make sure you have a lot of ChIPseq tracks in your database as else they will always be enriched in each analysis. For a cisTarget database you just need some input data that you can rank also make sure in case of ties that you randomize those rank assignment so you don't get artificial high rankings for your first regions.

**Comment by MatthewTCManion** [comment]
> @mtxellrb A script for creating a track database from bigWig TF ChIP-seq data is now added :`create_cistarget_track_databases.py`
> 
> https://github.com/aertslab/create_cisTarget_databases#create_cistarget_track_databasespy

Hello @ghuls , I am running into an issue using this script where the .bed file with regions to score is not recognized correctly, and I have tried a few different formats with no success. For reference, here is a screenshot of my most recent attempt to run the script, as well as the format of my .bed:


```
REGION_BED="/data/PetrosLab/Matt/scenicplus/chipseq/tracks/fwf_gene_assignments.bed"
DATABASE_PREFIX="CellType_750bp_with_binding"
SCRIPT_DIR="/data/PetrosLab/Matt/scenicplus/create_cisTarget_databases"
TRACKS_DIR="/data/PetrosLab/Matt/scenicplus/chipseq/tracks"
TRACK_LIST="track_names.txt"


"${SCRIPT_DIR}/create_cistarget_track_databases.py" \
	-b "${REGION_BED}" \
    -T "${TRACKS_DIR}" \
    -d "${TRACK_LIST}" \
    -o "${DATABASE_PREFIX}" \
    -t 20
```
![image](https://github.com/user-attachments/assets/3bca8b3d-dffc-45c7-9b47-72d6e9dc6013)

I assume the issue is with the format of the .bed or the genes/regions data, but I can't find what the proper format should be.

Thanks,
Matt
